zndraw/modify/__init__.py

```diff
@@ -105,6 +105,7 @@ class Move(UpdateScene):
     discriminator: t.Literal["Move"] = Field("Move")
 
     steps: int = Field(10, ge=1)
+    delay: float = Field(0.0, ge=0, description="Delay between steps in seconds")
```

Comment:
Both steps and delay should have a maximum value, otherwise you can crash the server by setting many steps and high delay

Reply:
This was for testing only - I want to remove this,